test/src/test/java/org/corfudb/runtime/view/LayoutViewTest.java

```diff
@@ -303,4 +323,114 @@ public void getConsensusFromCurrentMembers3Nodes() throws Exception {
         assertThat(getLayoutServer(SERVERS.PORT_2).getCurrentLayout()).isEqualTo(newLayout);
         assertThat(getLayoutServer(SERVERS.PORT_3).getCurrentLayout()).isEqualTo(newLayout);
     }
+
+    /**
```

Comment:
This is a great test.
I think we need one more test for a 3 node scenario where the quorum got filled up with the rank1 layout only.